Xamarin.Forms.Core/Shell/Shell.cs

```diff
@@ -162,8 +162,6 @@ static void OnShellColorValueChanged(BindableObject bindable, object oldValue, o
 
 		static readonly BindablePropertyKey MenuItemsPropertyKey =
 			BindableProperty.CreateReadOnly(nameof(MenuItems), typeof(MenuItemCollection), typeof(Shell), null, defaultValueCreator: bo => new MenuItemCollection());
-			
-#region IShellController
```

Comment:
Why drop the regions?

Reply:
for a few reasons:
- I don't like them
- they're not consistent with the rest of the code, or even the rest of the shell code
- they enforce some weird code ordering (this one had a region for BP, then a region for interface implementation. those 2, for example, can collide)
- they're out of date faster than code comments

now, I get why one might like them: being able to collapse region. I know a thing or two about where to place opening braces, that saves SO MUCH vertical space that collapsing is no longer required AT ALL.

Reply:
Fair enough. I didn't realize they weren't being applied consistently. 

> I know a thing or two about where to place opening braces, that saves SO MUCH vertical space that collapsing is no longer required AT ALL.
 
:)